src/components/account/edit.vue

```diff
@@ -1,5 +1,5 @@
 <!--
-Copyright 2017 ODK Central Developers
+Copyright 2020 ODK Central Developers
```

Comment:
This file was not actually moved. This PR deleted `ProjectHome` and created `AccountEdit`. Those are both small components, but they are otherwise unrelated.